2019/meeting-2019-09-03.md

```diff
@@ -0,0 +1,156 @@
+    - Dan will start a hackmd for the agenda for the competition committee meeting (#3)
+    - Rob will schedule a brainstorming session (#4)
+    - Kier will request permission to create subteams on Github (#5)
+    - Kier will create Github repo for KTC minutes and tasks (#6)
+    - KTC will resolve whether they want to approve the kit team addition to the ops manual via email (#9)
+    - Tom will email the people who have signed up so far (#7)
+* Motion passing arrangement: See motion 0 below.
+* Should we form a temporary team for expeditious preparation for the upcoming Kickstart?
+    * It is worth noting that the CTC are intending to send us an email regarding this.
+    * They seem to want Python 3 for this year, which is likely infeasible at this short notice.
+    * A temporary team will allow us to continue focusing on our structure.
+    * Suggestions from Trustees of Kkickstart in early October (1st or 2nd Saturday). Exact date unconfirmed
+* Trustees have published a list of Key Dates (https://docs.google.com/document/d/1hh-ByVaFs3Cy_zUZVkipeLUNMmKfSrjY3IvgSCgTqBQ/edit)
+* Github permissions: See motion 1 below.
+* Next meeting: Need to change day!
+* Email trustees indicating that we are having a KTC brainstorming day (emphasising KTC only...)
+* ~~Should we begin drafting an email to volunteers explaining that we are here, and still forming?~~
+* AOB
+
+## Minutes
+
+* Action point review
+    * Kajetan will email competition committee (#2)
+        * This was superseded by the trustees' proposal for a joint meeting
+    * Dan will start a hackmd for the agenda for the competition committee meeting (#3)
+        * Done, Dan emailed around a link today
+    * Rob will schedule a brainstorming session (#4)
+        * Not done yet
+    * Kier will request permission to create subteams on Github (#5)
+        * In progress, turns out team maintainers don't work expected. Kier will carry on chasing this up with the org admins and trustees
+    * Kier will create Github repo for KTC minutes and tasks (#6)
+        * Done
+    * KTC will resolve whether they want to approve the kit team addition to the ops manual via email (#9)
+        * Rob to kick-start this discussion
+    * Tom will email the people who have signed up so far (#7)
+        * Not done yet
+* Motion 0: motion-passing arrangement
+    * Could use github for formalising approval process
+        * We'll consider this next meeting
+    * All in favour, motion passed
+* Trustee key dates
+    * "Present plans for next kit development cycle"
+        * We would like development itself to be continuous
+        * However this is about plans - what are we planning to work on over the next few months?
+    * All looks good
+    * KD to tell trustees it all looks good
+* Should non-team-members have permission to participate in discussions on team repos (e.g. non-KTC on KTC minutes repo)?
+    * Some people in favour of closed discussions for improved team cohesion
```

Comment:
Clarification on "closed discussions": Publicly readable, not publicly writable.